mozdownload/parser.py

```diff
@@ -89,5 +89,5 @@ def handle_data(self, data):
         if not self.active_url:
             return
 
-        if self.active_url == data:
+        if data in self.active_url:
```

Comment:
We need this change because the actual data can contain a final slash or not, while `active_url` always has it.